integration/app/test-demo/README.md

```diff
@@ -0,0 +1,79 @@
+# README - React Testing Demo
+
+The React Testing Demo is used to test the Amazon Chime SDK React Components Library.
+
+Different tests may require different testing demos. Creating a new testing demo for each test would add extra maintenance work. We don't like it.
+
+Following the example of [WebRTC samples](https://webrtc.github.io/samples/), we come up with the concept of a Mini Test App - a minimalist, single-page application containing only the necessary components for a single test. All Mini Test Apps are sub-pages of the React Testing Demo.
+
+## Guideline
+
+If you want to add tests for a new group of related features, build a new Mini Test App.
+
+* For example, we there is only a "Roster Test App". If you want to add tests for video features, you should build a "Video Test App".
+
+If there is already an existing Mini Test App for the features that you are working on, you should add onto the Mini Test App.
+
+* For example, there is only a "Roster Test App". If you want to add tests for a new `useRosterSize` hook which returns the size of the roster, you should improve the "Roster Test App" to cover that hook.
+
+## Run the React Testing Demo Locally
+
+1. If you have not configure your AWS credentials yet, do a [quick configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+2. Install the dependencies: `npm install`.
+3. Start the webpack server and node server concurrently: `npm start`.
+4. Open <http://localhost:9000/> in your browser.
+
+## Add a New Mini Test App
+
+1. Implement a new Mini Test App under `pages` folder.
+   * You could reuse the components and containers to facilitate your development.
+
+2. Create a new route for Mini Test App in `src/app.tsx`.
+
+    ```js
+    const App: React.FC = () => (
+      <ThemeProvider theme={lightTheme}>
```

Comment:
Can follow the existing code in tests but should we add where do we import ThemeProvider, BrowserRouter and so on from.